### PR TITLE
Resolve Grype CVE 

### LINKED ISSUE #890: Update dependencies to resolve Grype CVEs
## Is your feature request related to a problem? Please describe.
A number of steeltoe dependencies need updated to latest versions to resolve CVEs as reported by Grype

| Name | Installed | Fixed | Type | Vulnerability | Severity |
| ------------- | ------------- |------------- | ------------- |------------- | ------------- |
| Google.Protobuf | 3.11.4 | 3.15.0 | dotnet | GHSA-77rm-9x9h-xj3g | High |
| Microsoft.AspNetCore.Authentication.JwtBearer | 3.1.0 | 3.1.18 | dotnet | GHSA-q7cg-43mg-qp69 | Medium|
Microsoft.AspNetCore.Authentication.JwtBearer | 5.0.0 | 5.0.9 | dotnet | GHSA-q7cg-43mg-qp69 | Medium
Microsoft.AspNetCore.Authentication.JwtBearer | 2.1.0 | 2.1.29 | dotnet | GHSA-q7cg-43mg-qp69 | Medium
Microsoft.AspNetCore.Http | 2.1.0 | 2.1.22 | dotnet | GHSA-hxrm-9w7p-39cc | High
Microsoft.AspNetCore.Server.Kestrel.Core | 2.1.0 | 2.1.7 | dotnet | GHSA-6px8-22w5-w334 | High
Microsoft.AspNetCore.Server.Kestrel.Core | 2.1.0 | 2.1.2 | dotnet | GHSA-cgpw-2gph-2r9g | Medium
Microsoft.NETCore.App | 2.1.0 | 2.1.7 | dotnet | GHSA-5633-f33j-c6f7 | Medium
Microsoft.NETCore.App | 2.1.0 | 2.1.7 | dotnet | GHSA-6px8-22w5-w334 | High
Microsoft.NETCore.App | 2.1.0 | 2.1.8 | dotnet | GHSA-x5qj-9vmx-7g6g | Medium
Newtonsoft.Json | 9.0.1 | 13.0.1 | dotnet | GHSA-5crp-9r3c-p9vr | High
Newtonsoft.Json | 10.0.1 | 13.0.1 | dotnet | GHSA-5crp-9r3c-p9vr | High
Newtonsoft.Json | 11.0.2 | 13.0.1 | dotnet | GHSA-5crp-9r3c-p9vr | High
Newtonsoft.Json | 10.0.2 | 13.0.1 | dotnet | GHSA-5crp-9r3c-p9vr | High
Newtonsoft.Json | 12.0.2 | 13.0.1 | dotnet | GHSA-5crp-9r3c-p9vr | High
Newtonsoft.Json | 10.0.3 | 13.0.1 | dotnet | GHSA-5crp-9r3c-p9vr | High
Newtonsoft.Json | 12.0.3 | 13.0.1 | dotnet | GHSA-5crp-9r3c-p9vr | High
SSH.NET | 2020.0.0-beta1 | 2020.0.2 | dotnet | GHSA-72p8-v4hg-v45p | Medium
System.DirectoryServices.Protocols | 4.7.0 | 5.0.1 | dotnet | GHSA-9cxh-gqpx-qc5m | Medium
System.IO.Pipelines | 4.5.0 | 4.5.1 | dotnet | GHSA-j378-6mmw-hqfr | High
System.Net.Http | 4.3.0 | 4.3.4 | dotnet | GHSA-7jgj-8wvc-jh57 | High
System.Net.Http.WinHttpHandler | 4.0.0 | 4.0.1 | dotnet | GHSA-qhqf-ghgh-x2m4 | High
System.Net.Http.WinHttpHandler | 4.0.0 | 4.0.1 | dotnet | GHSA-ch6p-4jcm-h8vh | Medium
System.Net.Http.WinHttpHandler | 4.0.0 | 4.0.1 | dotnet | GHSA-j8f4-2w4p-mhjc | Medium
System.Net.Http.WinHttpHandler | 4.0.0 | 4.0.1 | dotnet | GHSA-6xh7-4v2w-36q6 | High
System.Net.Security | 4.3.0 | 4.3.1 | dotnet | GHSA-qhqf-ghgh-x2m4 | High
System.Net.Security | 4.3.0 | 4.3.1 | dotnet | GHSA-6xh7-4v2w-36q6 | High
System.Net.Security | 4.3.0 | 4.3.1 | dotnet | GHSA-j8f4-2w4p-mhjc | Medium
System.Net.Security | 4.3.0 | 4.3.1 | dotnet | GHSA-ch6p-4jcm-h8vh | Medium
System.Private.ServiceModel | 4.5.0 | 4.5.3 | dotnet | GHSA-p9wx-v264-q34p | Medium
System.ServiceModel.Primitives | 4.5.0 | 4.5.3 | dotnet | GHSA-p9wx-v264-q34p | Medium
System.Text.Encodings.Web | 4.6.0 | 4.7.2 | dotnet | GHSA-ghhp-997w-qr28 | Critical
System.Text.Encodings.Web | 4.5.0 | 4.5.1 | dotnet | GHSA-ghhp-997w-qr28 | Critical
System.Text.Encodings.Web | 4.7.0 | 4.7.2 | dotnet | GHSA-ghhp-997w-qr28 | Critical
System.Text.Encodings.Web | 4.4.0 | 4.5.1 | dotnet | GHSA-ghhp-997w-qr28 | Critical
System.Text.Encodings.Web | 4.7.1 | 4.7.2 | dotnet | GHSA-ghhp-997w-qr28 | Critical
System.Text.RegularExpressions | 4.3.0 | 4.3.1 | dotnet | GHSA-cmhx-cq75-c4mj | Medium

### PR DESCRIPTION
**Pull request by hananiel**
Fixes #890

### CODE REVIEW

File: src/Common/src/Common/Steeltoe.Common.csproj [hunk 1]
```diff
@@ -21,6 +21,12 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="$(ExtensionsVersion)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="$(JsonNetVersion)" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```
Comment (TimHess) [inline src/Common/src/Common/Steeltoe.Common.csproj:26]:
```suggestion
    <PackageReference Include="System.Net.Http" Version="$(SystemNetHttpVersion)" />
```

File: src/Common/src/Common/Steeltoe.Common.csproj [hunk 2]
```diff
@@ -21,6 +21,12 @@
...
+    <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
```
Comment (TimHess) [inline src/Common/src/Common/Steeltoe.Common.csproj:27]: Add to verstions.props

**Comment by hananiel** [comment]
> @hananiel : Did we not have any references in Steeltoe to the two packages with CVEs? Did you get a copy of his application/solution that he was having problems with? I asked him in slack for a copy of the app! Was it from App Accelerator? Do we just need to update it?
@dtillman I saw the one with System.Net.Http and addressed it. After going through the list of CVEs systematically addressing each one, the changes you see are what I had to make. I did not see a need to update System.Private.Uri. I don't have a copy of the app either.

**Comment by TimHess** [comment]
Thanks @hananiel for doing the work to define what it would take to resolve the CVE complaints. For anyone reading through these comments, these changes have been confirmed to resolve vulnerability scanner complaints that appear in applications using Steeltoe 3.2.0. However, adding these references (to override dependencies specified in packages that are upstream of Steeltoe) is a slippery slope towards an unmaintainable situation. As it sits today, we would need to force a two-major-version update of Newtonsoft (from v11 to v13), which is too risky of a change to make in a patch release. As a team, we've agreed that the best solution is for applications using Steeltoe to update these packages on their own.

**Pull request closed without merging**